storage/inchi/src/main/java/org/openscience/cdk/inchi/JniInChIInputAdapter.java

```diff
@@ -98,10 +101,23 @@ private static String checkOptions(final String ops) throws JniInchiException {
             } else if (isTimeoutOptions(op)) {
                 // only reformat if we actually have a decimal
                 if (isSubSecondTimeout(op)) {
-                    // because the JNI-InChI library is expecting an platform number, format it as such
-	                Double time = Double.parseDouble(op.substring(1));
-	                DecimalFormat format = new DecimalFormat("#.##");
-	                sbOptions.append(FLAG_CHAR).append('W').append(format.format(time));
+                    DecimalFormat format = new DecimalFormat("#.##", DecimalFormatSymbols.getInstance(Locale.ROOT));
+                    if (op.contains(",")) { // we accept ',' as decimal separator
+                        final DecimalFormatSymbols decimalFormatSymbols = format.getDecimalFormatSymbols();
+                        decimalFormatSymbols.setGroupingSeparator('\'');
+                        decimalFormatSymbols.setDecimalSeparator(',');
+                        format.setDecimalFormatSymbols(decimalFormatSymbols);
+                    }
+                    double time;
+                    try {
+                        time = format.parse(op.substring(1)).doubleValue();
+                        if (time <= 0.0) {
+                            throw new JniInchiException("Timeout value must be greater than 0.");
+                        }
+                    } catch (ParseException e) {
+                        throw new NumberFormatException(op.substring(1));
+                    }
+                    sbOptions.append(FLAG_CHAR).append('W').append(time);
```

Comment:
this will format the option in a basic number format, eg. `W0,01` will be sent to JniInchI like this: `W0.01`